packages/astrocite-eutils/src/eutils.ts

```diff
@@ -24,14 +24,32 @@ const FIELD_TRANSFORMS = new Map<
         ({ authors }) => {
             return {
                 author: authors
-                    .map(({ name }) => {
-                        const [family, given] = name.split(' ');
-                        return {
-                            family: family || '',
-                            ...(given ? { given } : {}),
-                        };
+                    .map(({ authtype, name }) => {
+                        switch (authtype) {
+                            case 'CollectiveName':
+                                if (!name) {
+                                    return null;
+                                }
+
+                                return {
+                                    literal: name,
+                                };
+
+                            case 'Author':
+                            default:
+                                const [family, given] = name.split(' ');
+
+                                if (!family) {
+                                    return null;
+                                }
+
+                                return {
+                                    family,
+                                    ...(given ? { given } : {}),
+                                };
+                        }
                     })
-                    .filter(({ family }) => family !== ''),
+                    .filter(item => item !== null) as Person[],
```

Comment:
I don't like using `null` ever if I can avoid it since `null` generally makes things more difficult to deal with and doesn't really offer any benefit over `undefined` 9 times out of 10.

What about keeping the majority of these changes, but instead returning `undefined` and then in the filter, just do a `.filter(Boolean)` to pluck out all the `undefined`s?

Reply:
Sounds good - updated to use `undefined`.